doc/api/index.md

```diff
@@ -169,6 +170,54 @@ player.loadVideo({
 });
 ```
 
+<a name="meth-reload"></a>
+### reload #####################################################################
+
+_arguments_:
+  - _options_ (``Object``)
+
+Re-load the last loaded content, with its original loadVideo options, as fast as
+possible, using the last loaded manifest. The aim is to reduce loading time by
+removing the fetching and parsing steps.
+
+At each new loadVideo call, the player fetches the manifest, and stores it as
+the new last content manifest.
+
+This API can be called at any time after a content has been loaded (the LOADED
+state has been reached), even if the player has been stopped since and even if
+it was due to a fatal error.
+
+The user may need to call this API in several cases. For example, it may be used
+in case of an error that will not reproduce or inversely when the error is
+consistent at a certain playback time (e.g. due to a specific chunk).
+
+The options argument is an object containing :
+- _startAt_ (``Object | undefined``): same as the loadVideo startAt argument 
+[loadVideo function](#meth-loadVideo). If it is not defined, the startAt from
+the last loaded content will be used.
```

Comment:
I've been thinking about the `the startAt from the last loaded content will be used` part.
Is it always what we want?

For example, what if the original `loadVideo` call had a `startAt` set, but now we don't want one anymore?

Reply:
Indeed, maybe we can consider that the RxPlayer user knows what startAt had been set previously. The new comportment can be : if no startAt, erase the startAt from the first load video call.

Reply:
I changed it into : if options are set, consider startAt to erase the last content options startAt. If there are not, keep original options values.

Reply:
Wouldn't that be too complex for the user?

Like if we add other options, adding an unrelated one would unexplicably change the `startAt` behavior when compared to not setting that option.

Reply:
We could make a difference between the property being set to undefined, and the property not being defined.